executor/stale_txn_test.go

```diff
@@ -1302,23 +1302,17 @@ func TestPlanCacheWithStaleReadByBinaryProto(t *testing.T) {
 	tk.MustExec("use test")
 	tk.MustExec("create table t1 (id int primary key, v int)")
 	tk.MustExec("insert into t1 values(1, 10)")
+	time.Sleep(time.Second)
 	se := tk.Session()
-	tk.MustExec("set @a=now(6)")
-	time.Sleep(time.Millisecond * 5)
+	now := time.Now()
+	time.Sleep(time.Second * 2)
```

Comment:
Maybe the following change is enough:

```
se := tk.Session()
time.Sleep(time.Millisecond * 100)
tk.MustExec("set @a=now(6)")
tk.MustExec("update t1 set v=100 where id=1")
```

See: https://github.com/pingcap/tidb/pull/32959/files#diff-6f8c739b7faa2c14e5a2e28f2a18abfe12ffe1bf45d4bed9add7dfec69b37710R118-L120 . Only add a sleep before `tk.MustExec("set @a=now(6)")` and 100ms is enough.